modules.d/35network-manager/module-setup.sh

```diff
@@ -38,6 +38,12 @@ install() {
         inst "$dbussystem"/org.freedesktop.NetworkManager.conf
         inst_multiple nmcli nm-online
 
+        # teaming support under systemd+dbus
+        inst_multiple -o \
+            "$systemdsystemunitdir"/teamd@.service \
```

Comment:
Currently, NetworkManager spawns teamd directly and then waits that teamd appears on D-Bus to communicate with it.
This line above is not needed because teamd is not started as a systemd service.

The rest LGTM, I also tried the patch and it works well. Thanks!

Reply:
fixed in latest push. Thanks @bengal.

Reply:
LGTM now.